core/ExceptionHandlerDefaults.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+
+/**
+ * Provides a centralized location to set Exception Handling defaults.
+ * Can be overwritten to apply throughout the application.
+ */
+export class ExceptionHandlerDefaults {
+
+    /**
+     * Property paths within error details JSON to replace with REDACT_TEXT
```

Comment:
Am trying to see about getting these on to ExceptionHandler itself for API simplicity.  Would like the exception package *not* to depend on core.  So we can go back to your original method of passing the redaction to the stringify method, or perhaps just remove the stringify method from the public API.